docs/adr/adr-0022-new-icon-system.md

```diff
@@ -242,6 +242,8 @@ This simplifies the API for developers because:
 
 Where Remix Icon is a great library, it seems to be currently on [pause](https://github.com/Remix-Design/RemixIcon/issues/232).
 
+_edit (2023-09-15)_: actually, the Remix Icon project [resumed its activity](https://github.com/Remix-Design/RemixIcon/issues/232#issuecomment-1017140063) since early 2023! 🎉
```

Comment:
question: I was wondering if this link was more appropriate: `https://github.com/Remix-Design/RemixIcon/issues/232#issuecomment-1492911872`

Reply:
Thanks both of you! I'll leave it as is for now. :pray: